README.md

```diff
@@ -0,0 +1,22 @@
+## Jupyter ![](https://img.shields.io/github/forks/TheAlgorithms/Jupyter?style=social) ![](https://img.shields.io/github/stars/TheAlgorithms/Jupyter?style=social) ![](https://img.shields.io/github/watchers/TheAlgorithms/Jupyter?style=social) <br>
+
+![](https://img.shields.io/github/repo-size/TheAlgorithms/Jupyter) ![](https://img.shields.io/github/downloads/TheAlgorithms/Jupyter/total)<br>
+![](https://img.shields.io/github/issues/TheAlgorithms/Jupyter?color=green) ![](https://img.shields.io/github/issues-pr/TheAlgorithms/Jupyter?color=green) ![](https://img.shields.io/github/last-commit/TheAlgorithms/Jupyter) ![](https://img.shields.io/github/contributors/TheAlgorithms/Jupyter)
+
+## Clone git repository
+
+```sh
+    $ Git clone "https://github.com/TheAlgorithms/Jupyter.git"
+```
+
+You can run and edit the algorithms or contribute to them using [Gitpod.io](https://www.gitpod.io/), a free online development environment, with a single click.
+
+[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](http://gitpod.io/#https://github.com/TheAlgorithms/Jupyter)
+
+## Contributing
+
+Please read [Contributing.md](./CONTRIBUTION.md/Contributing.md) for information on how to contribute to Jupyter.
```

Comment:
Let's remove the `CONTRIBUTION.md` directory and put `Contributing.md` at the root.

Reply:
Modified sir 👍